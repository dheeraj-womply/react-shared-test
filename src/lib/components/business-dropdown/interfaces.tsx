export interface IBusinessDropdown {
    data: IBusinessDropdownData;
    handleChange: (args: any) => void;
}

export interface IBusinessDropdownData {
    businessDropdown: BusinessLocation[],
    activeLocation: BusinessLocation | undefined
}

export interface Address {
    address1: string;
    address2: string;
    city: string;
    state: string;
    zip: string;
}

export interface Category {
    id: number;
    name: string;
}

export interface Coordinates {
    latitude: number | null;
    longitude: number | null;
}

export interface Analytics {
    trial: boolean;
    is_trial: boolean;
    trial_end_date: string | null;
    claimed_at: string | null;
    claimed_at_month: string | null;
    sales_rep_id: string | null;
}

export interface EmailSettings {
    reputation_alerts: boolean;
    monthly_business_summaries: boolean;
    weekly_business_summaries: boolean;
    weekly_competitor_summaries: boolean;
}

export interface BusinessLocation {
    id: number;
    slug: string;
    name: string;
    business_source: string;
    business_phone: string;
    website_url: string;
    price: number | null;
    address: Address;
    category: Category;
    coordinates: Coordinates;
    analytics: Analytics;
    email_settings: EmailSettings;
}