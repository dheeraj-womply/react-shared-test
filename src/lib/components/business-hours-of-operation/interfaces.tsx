export interface IDay {
    day: string,
    hours : Array<IHoursRange>,
    isClosed: boolean
}

interface IHoursRange {
    openTime: string | null,
    closeTime: string | null
}

export interface ITimepickerProps {
    handleDateChange:(date: string | null, fieldName: 'openTime'|'closeTime', index:number) => void,
    fieldName: 'openTime'|'closeTime',
    value: string | null,
    index: number,
    disabled?: boolean
}

export interface IBusinessDaysOfWeek {
    day: string;
    openTime: string | null;
    closeTime: string | null;
}