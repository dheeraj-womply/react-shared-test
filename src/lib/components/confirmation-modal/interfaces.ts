export interface ConfirmationModalComponent {
    isModalOpen?: boolean;
    isMobile?: boolean;
    title: React.ReactNode;
    content:  React.ReactNode;
    primaryButtonText: string;
    onPrimaryButtonClick: () => any;
    secondaryButtonText: string;
    onSecondaryButtonClick: () => any;
}
