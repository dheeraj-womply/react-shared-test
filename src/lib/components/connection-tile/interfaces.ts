import { MouseEvent } from 'react'

export enum TileStatus {
    CONNECTED_VALID = 'CONNECTED_VALID',
    CONNECTED_INVALID = 'CONNECTED_INVALID',
    NOT_CONNECTED = 'NOT_CONNECTED',
    UNABLE_TO_CONNECT = 'UNABLE_TO_CONNECT',
    CONNECTION_PENDING = 'CONNECTION_PENDING',
}

export interface IConnectionItem {
    label: string;
    icon?: string;
    status: TileStatus;
}

export interface IConnectionTile {
    tile: IConnectionItem;
    onClick?: (event?: MouseEvent<HTMLDivElement>) => void;
}