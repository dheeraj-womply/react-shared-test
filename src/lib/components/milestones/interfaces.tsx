
export interface IMilestoneBubble {
    milestonesData: IMilestonesData;
    onClick: (item: IMilestone) => void;
}

export interface IMilestoneProgressCircle {
    milestonesData: IMilestonesData;
}

export interface IMilestonesData {
    milestones: IMilestone[];
    milestoneCount: number;
    completedMilestoneCount: number;
    pollingPages: IPollingPage[]
}

export interface IMilestone {
    key: string;
    description: string;
    displayText: string;
    urlPage: string;
    urlSubDomain: string;
    sortIndex: number;
    productId: number;
    completedTime: number;
}

export interface IPollingPage {
    milestoneKey: string;
    urlPage: string;
    urlSubDomain: string
}