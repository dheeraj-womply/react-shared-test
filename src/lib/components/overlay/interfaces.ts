export interface OverlayComponent {
  children?: React.ReactNode
  show?: boolean
  allowUserActions?: boolean
  hideSpinner?: boolean
  spinnerTop?: string
  rootStyles?: string
}
