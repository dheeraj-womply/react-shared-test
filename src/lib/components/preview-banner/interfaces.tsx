export interface IBannerProps {
    bannerId?: number;
    businessLocationId?: number;
    bannerAlignment?: string;
    displaySequence?: string|number;
    bannerType: string;
    isEnabled?: boolean;
    cssStyles: string; 
    startDate?: Date;
    endDate?: Date;
    bannerStart?: string;
    bannerFields: Array<IBannerFields>;
}

export interface IBannerFields {
    bannerFieldId: string;
    bannerFieldName: string;
    bannerFieldValue: string;
    bannerType: string;
    cssStyles: string;
    clickToAction?: string;
}