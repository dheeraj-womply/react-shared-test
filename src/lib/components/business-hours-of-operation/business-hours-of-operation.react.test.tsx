import React from 'react';
import renderer from 'react-test-renderer';
import { render, fireEvent } from '@testing-library/react';
import { BusinesHoursOfOperation } from '.';
import { IBusinessDaysOfWeek } from './interfaces';
import { screen } from '@testing-library/dom'

const days:Array<IBusinessDaysOfWeek> = [
    {day: 'MONDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'MONDAY', openTime: '04:00 PM', closeTime: '07:00 PM'},
    {day: 'TUESDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'WEDNESDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'THURSDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'FRIDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'SATURDAY', openTime: '07:00 AM', closeTime: '03:00 PM'},
    {day: 'SUNDAY', openTime: null, closeTime:null}
];
test('Business Hours of operation component renders as expected with data', () => {
    const component = renderer.create(
        <BusinesHoursOfOperation 
            businessDays={days}
            />
    );
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
});


test('Business Hours of operation component renders as expected without data', () => {
    const component = renderer.create(
        <BusinesHoursOfOperation />
    );
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
});


test('Open popover and click cancel button', async () => {
    const { findByTestId, container, findAllByTestId } = render(
        <BusinesHoursOfOperation 
            businessDays={days}
            />
    );
    const elements = await findAllByTestId('edit-hours');
    fireEvent.click(elements[0]);
    const btn = screen.getByTestId('cancel')
    btn.onclick = jest.fn();
    fireEvent.click(btn);
    expect(btn.onclick).toBeCalled()
});


test('Open popover, click checkbox for "closed" and click OK button', async () => {
    const { container, findAllByTestId, getByLabelText  } = render(
        <BusinesHoursOfOperation 
            businessDays={days}
            />
    );
    const elements = await findAllByTestId('edit-hours');
    fireEvent.click(elements[0]);
    
    const elem = getByLabelText('Closed');
    fireEvent.click(elem); 
    
    const btn = screen.getByTestId('save')
    btn.onclick = jest.fn();
    fireEvent.click(btn);
    
    expect(btn.onclick).toBeCalled()
});