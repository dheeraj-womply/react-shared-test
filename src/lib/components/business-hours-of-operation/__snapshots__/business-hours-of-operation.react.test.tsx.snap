// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business Hours of operation component renders as expected with data 1`] = `
<div
  className="sc-AxjAm kBuBUH"
>
  <div
    className="MuiGrid-root makeStyles-paper-1 MuiGrid-item MuiGrid-grid-xs-12"
    style={
      Object {
        "paddingLeft": 0,
      }
    }
  >
    <div
      className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
    >
      <div
        className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
        color="rgba(0, 0, 0, 0.87)"
        textalign="left"
      >
        Monday
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                color="rgba(0, 0, 0, 0.87)"
                textalign="center"
              >
                 - 
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
              customcolor="rgba(0, 0, 0, 0.87)"
              data-testid="edit-hours"
              float="none"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-entered"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Tuesday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Wednesday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Thursday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Friday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Saturday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Sunday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Business Hours of operation component renders as expected without data 1`] = `
<div
  className="sc-AxjAm kBuBUH"
>
  <div
    className="MuiGrid-root makeStyles-paper-1 MuiGrid-item MuiGrid-grid-xs-12"
    style={
      Object {
        "paddingLeft": 0,
      }
    }
  >
    <div
      className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
    >
      <div
        className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
        color="rgba(0, 0, 0, 0.87)"
        textalign="left"
      >
        Monday
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
          >
            <div
              className="MuiGrid-root MuiGrid-container"
            >
              <div
                className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                color="rgba(0, 0, 0, 0.87)"
                textalign="center"
              >
                 - 
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
              customcolor="rgba(0, 0, 0, 0.87)"
              data-testid="edit-hours"
              float="none"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiCollapse-container MuiCollapse-entered"
      style={
        Object {
          "minHeight": "0px",
        }
      }
    >
      <div
        className="MuiCollapse-wrapper"
      >
        <div
          className="MuiCollapse-wrapperInner"
        >
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Tuesday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Wednesday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Thursday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Friday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Saturday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root sc-AxirZ lgjYw MuiGrid-container"
          >
            <div
              className="MuiGrid-root sc-AxhCb eUHVRl MuiGrid-item MuiGrid-grid-xs-4"
              color="rgba(0, 0, 0, 0.87)"
              textalign="left"
            >
              Sunday
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
            >
              <div
                className="MuiGrid-root MuiGrid-container"
              >
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
                >
                  <div
                    className="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      className="MuiGrid-root sc-AxhCb ihfzax MuiGrid-item MuiGrid-grid-xs-12"
                      color="rgba(0, 0, 0, 0.87)"
                      textalign="center"
                    >
                       - 
                    </div>
                  </div>
                </div>
                <div
                  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
                  onClick={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                    }
                  }
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root sc-AxiKw cA-dXMf"
                    customcolor="rgba(0, 0, 0, 0.87)"
                    data-testid="edit-hours"
                    float="none"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
